### ISSUE TITLE
just thank you！

### ISSUE THREAD

**Issue by Myheartletmego** [Z4ee/Pipsi-SR-Releases#102]
There may be many people who want to update to the latest 2.6 version, but I want to say thank you for the free updates you have provided that have improved the game experience.
Some people have also mentioned wanting to add a feature that multiplies the effectiveness of certain actions, but I strongly advise against adding such unbalanced features. This would disrupt the game's balance and ultimately affect the revenue of the game developers, which could ultimately result in Pipsi being blocked.
In conclusion, I hope you keep up the good work. I am also very interested in learning about scripting and development, but my skills are not yet sufficient.
